### PR TITLE
you're welcome

### PR DESCRIPTION
**Pull request by RedTechEngineer**
dont you dare reject this rat girl

### CODE REVIEW

**Comment by brochacho6** [comment]
how did you do at kidnapping the femoid redtech?

**Comment by RedTechEngineer** [comment]
> how did you do at kidnapping the femoid redtech?

it went well. infact, i got two for the price of one!

**Review by elonmusksama — COMMENTED** [review]
😹

File: index.html [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-    <script type="text/javascript" src="js/TitleScrollTroll.js"></script>
-    <!-- Open Graph stuff  -->
-    <meta name="og:title" content="PVG">
-    <meta name="og:description" content="PVG Official Website">
-    <meta name="og:image" content="https://b.catgirlsare.sexy/qKNm_k07.png">
-    <meta name="og:url" content="https://proudvirgingang.github.io/">
-    <meta name="og:site_name" content="proud virgin gang">
-    <meta name="og:locale" content="en_US">
-    <meta name="og:type" content="website">
+	<meta charset="UTF-8">
```
Comment (elonmusksama) [inline index.html:5]: 😹

File: index.html [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+	<meta name="og:locale" content="en_US"/>
+	<meta name="og:type" content="website"/>
 </head>
 <body>
-    <div class="container">
-        <img src="assets/pvg-shrew.png"/>
-        <h1 class="title">P(roud) V(irgin) G(ang)</h1>
-        <p class="text">Website for the Proud Virgin Gang | coming soon™</p>
-    </div>
+	<img class="head" src="assets/pvg-shrew.avif" alt="shrew"/>
```
Comment (elonmusksama) [inline index.html:24]: 😹

File: index.html [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
+	<h1>
+		<abbr title="Proud Virgin Gang">PVG</abbr></h1>
+	<h2>Website for the Proud Virgin Gang | coming soon&trade;</h2>
+	<p>RedTechEngineer edition, new and improved! This is clearly
```
Comment (elonmusksama) [inline index.html:28]: 🦍

**Pull request closed without merging**